lightning/src/routing/mod.rs

```diff
@@ -24,9 +24,13 @@ use sync::{Mutex, MutexGuard};
 ///
 ///	Scoring is in terms of fees willing to be paid in order to avoid routing through a channel.
 pub trait Score {
-	/// Returns the fee in msats willing to be paid to avoid routing through the given channel
-	/// in the direction from `source` to `target`.
-	fn channel_penalty_msat(&self, short_channel_id: u64, source: &NodeId, target: &NodeId) -> u64;
+	/// Returns the fee in msats willing to be paid to avoid routing `send_amt_msat` through the
+	/// given channel in the direction from `source` to `target`.
+	///
+	/// The channel's capacity in satoshis (given other HTLCs to be sent) is given by
+	/// `channel_capacity_sats`. It may be guessed from various sources or assumed from no data.
```

Comment:
Don't quite get the part "(given other HTLCs to be sent)". I think the first sentence can be removed?